### ISSUE TITLE
fix(aio): remove use of `System.import` from `PrettyPrinter`

### ISSUE THREAD

**Issue by petebacondarwin** [angular/angular#23700]
The latest CLI build now reports this as a problem:

```
WARNING in ./src/app/custom-elements/code/pretty-printer.service.ts
System.import() is deprecated and will be removed soon. Use import() instead.
For more info visit https://webpack.js.org/guides/code-splitting/
```

**Comment by gkalpak** [comment]
Related (but not the same): #21560

**Comment by IgorMinar** [comment]
I tried to do this already but I ran into cli/typescript issues back when cli wasn't stable yet. If I recall correctly it was a compilation error.

**Comment by filipesilva** [comment]
This is related to #21560 but indeed not the same. They key part is:

> Modern tooling is beginning to deprecate (notably Webpack 4 webpack/webpack#6321) System.import() detection in favor of the standard and it should not be unexpected that more follow suit.

Webpack 4 is deprecating `System.import()` support. We specialcase `@angular/core` to not show this warning, but it also has a `System.import()` there.

**Comment by th0r** [comment]
@filipesilva could you share the way you've silenced this warning in `angular-cli`?

**Comment by filipesilva** [comment]
@th0r this is what we did internally:

https://github.com/angular/devkit/blob/c535c8eac3841e2254bb8046e0f0ff8c3cb81c33/packages/angular_devkit/build_angular/src/angular-cli-files/models/webpack-configs/common.ts#L254-L259

**Comment by th0r** [comment]
Wow, interesting! Thanks.

**Comment by gkalpak** [comment]
Thishas been fixed in #32923.
